homework_6/package_2/Car.java

```diff
@@ -0,0 +1,19 @@
+package package_2;
+
+public class Car {
+    private void startElectricity(){
```

Comment:
Please apply formatting to all the files, IDEA can help you with that just press CTRL+ALT+L on a chosen class